src/restic/checker/checker.go

```diff
@@ -587,10 +587,6 @@ func (c *Checker) checkTree(id restic.ID, tree *restic.Tree) (errs []error) {
 	for _, node := range tree.Nodes {
 		switch node.Type {
 		case "file":
-			if node.Content == nil {
```

Comment:
Well, that would defeat the purpose of the change, as Content can now be legit nil.